packages/components/pages/f-account-info/src/components/_tests/AccountInfo.test.js

```diff
@@ -18,48 +36,106 @@ const createStore = ({ state = {}, actions = {} } = {}) => new Vuex.Store({
     }
 });
 
-let sutProps;
+const mountAccountInfo = ({
+    actions = storeActions,
+    state = storeState,
+    mocks = sutMocks,
+    propsData = sutProps,
+    data = dataDefaults,
+    storeOverride = null,
+    initialiseOverride = null
+} = {}) => {
+    const store = storeOverride || createStore({ state, actions });
+    store.registerModule = registerStoreModuleSpy;
+    store.hasModule = hasModuleSpy;
+
+    if (initialiseOverride) {
+        AccountInfo.methods.initialise = initialiseOverride;
+    }
```

Comment:
For this kind of pattern in JavaScript, I personally prefer this one-liner:
`AccountInfo.methods.initialise = initialiseOverride || AccountInfo.methods.initialise;`